plugins/english/foxaholic.js

```diff
@@ -44,76 +44,79 @@ async function parseNovelAndChapters (novelUrl){
     chapters: [],
   };
 
-  novel.name = loadedCheerio('.post-title > h1')
-    .text()
-    .replace(/[\t\n]/g, '')
-    .trim();
+  novel.name = loadedCheerio('.post-title > h1').text().trim();
 
   novel.cover = loadedCheerio('.summary_image > a > img').attr('data-src');
 
   loadedCheerio('.post-content_item').each(function () {
     const detailName = loadedCheerio(this)
       .find('.summary-heading > h5')
       .text()
-      .replace(/[\t\n]/g, '')
-      .trim();
-    const detail = loadedCheerio(this)
-      .find('.summary-content')
-      .text()
-      .replace(/[\t\n]/g, '')
       .trim();
+    const detail = loadedCheerio(this).find('.summary-content').html();
 
     switch (detailName) {
       case 'Genre':
-        novel.genre = detail.trim().replace(/[\t\n]/g, ',');
+        novel.genre = loadedCheerio(detail)
```

Comment:
`novel.genres`

same for the others

Reply:
i change all mention of ``genre`` to ``genres``